service/user/service.go

```diff
@@ -0,0 +1,120 @@
+package user
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/stasBigunenko/monorepa/pkg/storage/newStorage"
+	"unicode"
+
+	"github.com/google/uuid"
+	"github.com/stasBigunenko/monorepa/model"
+)
+
+type UsrService struct {
+	Us      model.UserHTTP
+	storage newStorage.NewStore
+}
+
+func NewUsrService(s newStorage.NewStore) *UsrService {
+	return &UsrService{
+		storage: s,
+	}
+}
+
+func (u *UsrService) Get(_ context.Context, id uuid.UUID) (model.UserHTTP, error) {
+
+	res, err := u.storage.Get(context.Background(), id)
+	if res == nil && err == nil || err != nil {
```

Comment:
Can split to error check and res check(2 different if). Also don`t check `err == nil || err != nil` - same conditions

Reply:
forgot brackets

Reply:
remade on two checks